Formula/biosig.rb

```diff
@@ -30,6 +30,8 @@ class Biosig < Formula
     sha256 "75df4a79b8d3d785942cbfd125ce45de49c3e7fa2cd19adb70caf8c4e30e13f0"
   end
 
+  patch :DATA
```

Comment:
Has this patch been applied upstream? Let's add a comment that explains when this patch can be removed, please. If there is a tracking issue we can link to, even better.

Reply:
This is fixed by PR #83191